templates/web/base/waste/bin_days_sidebar.html

```diff
@@ -21,7 +21,7 @@ <h3>Download your collection schedule</h3>
     [% END %]
 
        [% TRY %][% PROCESS waste/_more_services_sidebar.html %][% CATCH file %]
-       [% IF any_report_allowed OR any_request_allowed OR show_garden_subscribe %]
+       [% IF any_report_allowed OR any_request_allowed OR show_garden_subscribe OR (domestic_collection AND (c.user.from_body OR c.user.is_superuser)) %]
```

Comment:
This works fine - I think Brent is/will be the only thing using this default, it looks like all other existing Waste cobrands have created a custom `waste/_more_services_sidebar.html` template. So perhaps for consistency we should do the same here and not have a default, if they're all going to be slightly different.